docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -29,6 +29,18 @@ and then read metadata from each data file.
 Since Iceberg stores the paths to data files in the metadata files, it
 only consults the underlying file system for files that must be read.
 
+Requirements
+------------
+
+To use Iceberg, you need:
+
+* Iceberg table format compatible with Apache Iceberg format version 1.
+* Network access from the Trino coordinator and workers to the distributed
+  object storage.
+* Hive metastore service (HMS).
```

Comment:
Access to a...

Reply:
yeah .. we were grappling with that .. "Access to a" is kind of correct .. but lacks detail .. and its barely more useful as it stands. Any better suggestions @electrum or someone?

Reply:
For other things we are already using "network access from Trino coordinator to ...". That sounds fit to be used here IMO.

Reply:
@hashhar, all of those "other things" should be updated to use formal, standard English: 
   "network access from the Trino coordinator"

As for the original question, I'd be even more specific:
   "Access to a known working Hive metastore service (HMS)"
We don't want Iceberg customers to have problems because their HMS was set up cross-eyed.

Reply:
known working HMS is meaningless. 

Maybe we just go with 

Network access from the Trino coordinator to a Hive metastore service (HMS).


Question however.. do the workers also need access @hashhar .. I think yes..

Reply:
AFAIK from the code the workers don't need HMS access - but maybe @electrum or @phd3 can confirm.

Reply:
Yeah... I wasnt sure either but kinda recall that being mentioned..

Reply:
Applied suggestion:

> LGTM % `Network access from the Trino coordinator and workers to the HMS.` -> `Network access from the Trino coordinator to the HMS.`.
> 
> See https://github.com/trinodb/trino/pull/8214/files#r646879489